gamemodes/terrortown/gamemode/server/sv_player_ext.lua

```diff
@@ -228,7 +228,10 @@ plymeta.RemoveEquipmentWeapon = plymeta.StripWeapon
 -- @realm server
 function plymeta:SendEquipment(mode, itemName)
 	if not mode then
-		ErrorNoHalt("[TTT2] Define an EQUIPITEMS_mode for plymeta:SendEquipment(mode, itemName) to work.")
+		ErrorNoHalt("[TTT2] Define an EQUIPITEMS_mode for plymeta:SendEquipment(mode, itemName) to work.\n")
+
+		debug.Trace()
```

Comment:
This probably shouldn't be here :D

Reply:
This is intentional, because I got the error in the console and had no idea where it was called from. This way it is easier to debug imho

Reply:
why not use [ErrorNoHaltWithStack](https://wiki.facepunch.com/gmod/Global.ErrorNoHaltWithStack) in that case?

Reply:
Because I wasn't aware that this exists, thanks. I changed it